### PR TITLE
Insights Video List

### PR DESCRIPTION
**Pull request by revett**
### Problem

The video list does not exist...

### Solution

Create it using React components 👍 🎉 💯 

### Notes

- None

### CODE REVIEW

**Comment by revett** [comment]
<a href="https:&#x2F;&#x2F;trello.com&#x2F;c&#x2F;goEnRnjc&#x2F;335-sa-insights-video-list-page-as-reach-components-app"> [SA] Insights video list page as Reach components&#x2F;app</a>

**Comment by revett** [comment]
@benjdlambert can you review this please?

File: source/jsx/brands/insights/posts_table.jsx [hunk 1]
```diff
@@ -0,0 +1,73 @@
...
+  render() {
+    return (
+      <table className="fb-in-videos">
+        <thead>
+          <tr>
+            <th>Published</th>
+            <th>Video</th>
+            <th>
+              Reach{' '}
+              <span className="fb-in-info icon-info">
```
Comment (benjdlambert) [inline source/jsx/brands/insights/posts_table.jsx]: Are we wanting to make these seperate components?

File: source/jsx/brands/insights/posts_table.jsx [hunk 2]
```diff
@@ -0,0 +1,73 @@
...
+        </thead>
+        <tbody>
+          {this.props.rows}
+        </tbody>
+      </table>
+    );
+  }
+}
+
+InsightsPostsTable.propTypes = {
```
Comment (benjdlambert) [inline source/jsx/brands/insights/posts_table.jsx]: Can we use the `static` pattern from the gist?

File: source/jsx/brands/app.jsx [hunk 3]
```diff
@@ -9,7 +13,11 @@ export default class Brands extends React.Component {
   }
   render() {
     return (
-      <InsightsDashboard {...this.props} />
+      <Router history={browserHistory}>
```
Comment (benjdlambert) [inline source/jsx/brands/app.jsx]: Nice.

File: source/jsx/brands/insights/posts_table.jsx (hunk 2 above)
Comment (revett) [inline source/jsx/brands/insights/posts_table.jsx]: 👍

File: source/jsx/brands/insights/posts_table.jsx (hunk 1 above)
Comment (revett) [inline source/jsx/brands/insights/posts_table.jsx]: I could create it as a `TableHeader` component?

**Pull request merged**